### PR TITLE
 Created windows implementation of SerialShark

### PR DESCRIPTION
**Pull request by L0laapk3**

### CODE REVIEW

File: extras/SerialSharkWindows.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            except Exception as e:
+                print("[!] Please enter a number!")
+                continue
+            canBreak = True
+except KeyboardInterrupt:
+    print("\n[+] Exiting...")
+    exit()
+    
+try:
+    wiresharkCmdInput = raw_input("[?] Select wireshark location (default 'D:\Program Files\Wireshark\Wireshark.exe'): ")
```
Comment (spacehuhn) [inline extras/SerialSharkWindows.py]: `D:\Program Files\Wireshark\Wireshark.exe` 
What if I have wireshark installed somewhere else?

**Comment by spacehuhn** [comment]
nice work!

**Comment by L0laapk3** [comment]
Im gonna be completely honest here. I've created this over 1 year ago and I only yesterday accidentally typed commands in the wrong git and ended up creating this pull request. By all means, do use it if you like it or make any modifications to it, but I don't know what it does anymore xD from what I remember it did work for me :p I *think* rawinput lets you set the install location, the default location is just if you leave it blank

**Comment by L0laapk3** [comment]
Ok, I've figured it out :P a30c4ed9964e1c32f5bb56ad823ef8cfdc46b149 I ment to commit to another git and then merged it straight to master :P I'll cancel the pull request, but if you're interested in merging this you're still free to do that :p the readme needs some work tho

**Comment by spacehuhn** [comment]
Yeah alright, I haven't touched this repo in ages anyway...
Thanks for your work anyway

**Pull request closed without merging**